worldedit-core/src/main/java/com/fastasyncworldedit/core/history/changeset/AbstractChangeSet.java

```diff
@@ -346,6 +354,8 @@ public void add(int x, int y, int z, int combinedFrom, BaseBlock to) {
         Runnable wrappedTask = () -> {
             try {
                 writeTask.run();
+            } catch (Throwable t) {
+                t.printStackTrace();
```

Comment:
This should use a logger and in best case not catch errors that can't be ignored

Reply:
Without this, anything thrown in the writeTask is not necessarily made visible in the console - something I came across whilst trying to work out why history wasn't working

Reply:
What do you think about logging the stacktrace and rethrowing? I'd prefer that personally

Reply:
Rethrowing would only make sense if completenow is false, and probably in that case shouldn't be printed here in the first place. Rethrowing if it's true will do nothing because this is the too of the task chain so nothing else is going to be done as the runnable is submitted unblockingly